OutOfSchool/OutOfSchool.WebApi/Models/PhotoDto.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.ComponentModel.DataAnnotations;
+using OutOfSchool.Services.Enums;
+
+namespace OutOfSchool.WebApi.Models
+{
+    public class PhotoDto
+    {
+        public long Id { get; set; }
+
+        public string Path { get; set; }
+
+        [Required]
+        public long EntityId { get; set; }
+
+        [Required]
+        public EntityType EntityType { get; set; }
+
+        [Required]
+        public PhotoExtension PhotoExtension { get; set; }
+
+        public byte[] Photo { get; set; }
```

Comment:
Byte array image in JSON will look like an array with **millions** of entries :) 
![image](https://user-images.githubusercontent.com/24808838/127285953-70cacfc5-fe35-4e5c-8a56-4ce3805198a2.png)
if we stick to JSON then use a **Base64** string. But better use [multipart/form-data](https://docs.microsoft.com/en-us/aspnet/core/mvc/models/file-uploads?view=aspnetcore-3.0#upload-large-files-with-streaming) as IFormFile has some additional info for easy validation.